images/1try/index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>twitter</title>
+
+    <link rel="stylesheet" href="Scontainer.css">
+    <link rel="stylesheet" href="left.css">
+
+</head>
+
+<body>
+
+    <!-- main layout -->
```

Comment:
Using the comments to separate which code is what is NICE! It's easier to read the code